umbral/umbral.py

```diff
@@ -95,13 +174,40 @@ def reconstruct(self):
         
         return ReconstructedCapsule(e_prime=e, v_prime=v, x=cfrag_0.point_eph_ni)
 
+    def __bytes__(self):
+        self.to_bytes()
+
 
 class ReconstructedCapsule(object):
     def __init__(self, e_prime, v_prime, x):
         self.e_prime = e_prime
         self.v_prime = v_prime
         self.point_eph_ni = x
 
+    @classmethod
+    def from_bytes(self, data: bytes, curve):
+        """
+        Instantiate ReconstructedCapsule from serialized data.
+        """
+        e_prime = Point.from_bytes(data[0:33], curve)
+        v_prime = Point.from_bytes(data[33:66], curve)
+        eph_ni = Point.from_bytes(data[66:99], curve)
+
+        return ReconstructedCapsule(e_prime, v_prime, eph_ni)
+
+    def to_bytes(self):
+        """
```

Comment:
Again, I'm left wondering *why* these attrs represent a complete repr.  If I wanted to understand the essence of bytes representation of this class, I'd look here.  So let's add a bit of docstring explaining it.